nginx/Dockerfile

```diff
@@ -41,12 +42,15 @@ RUN ./configure \
 RUN make -j$(nproc)
 RUN make install
 
-# * Use Ubuntu as we need to install several pieces of software
-FROM ubuntu:latest
+# * Use Ubuntu to run Nginx
+FROM ubuntu:latest AS final
+
+RUN apt-get update \
+	&& apt-get install -y \
+	libpcre3 \
+	&& apt-get clean
```

Comment:
Is this required? This is running fine without this, and I'd like to keep the images as light as possible

Reply:
Yes, it's required. Otherwise, nginx crashes:

```
nginx: error while loading shared libraries: libpcre.so.3: cannot open shared object file: No such file or directory
```

However, it's only required because you specified `--with-pcre` to the Nginx configure script. I kept it because I assumed you were using the Perl compatible regular expressions in the library, but if it isn't required it can be removed.

Reply:
In my tests I was unable to build nginx at all without it, but I'm not using it myself no. We can play around with this

Reply:
Looks like the HTTP rewrite module is the thing that requires the PCRE. Okay to disable that?